lib/quantcast/index.js

```diff
@@ -156,38 +194,37 @@ Quantcast.prototype.completedOrder = function(track){
 };
 
 /**
- * Generate quantcast labels.
- *
- * Example:
- *
- *    options.advertise = false;
- *    labels('event', 'my event');
- *    // => "event.my event"
+ * Helper method to generate quantcast labels.
  *
- *    options.advertise = true;
- *    labels('event', 'my event');
- *    // => "_fp.event.my event"
- *
- * @param {String} type
- * @param {String} ...
+ * @param {String} prefix
+ * @param {Array} pageMeta
```

Comment:
What I'm really saying is, we should improve the documentation here. The questions I have about this line are:

- What does `pageMeta` represent?
- Is there documentation you can link to here that would help a future developer (me!) not have to do the digging you had to?
- Can you give one or a few examples?<sup>†</sup> I notice we removed the old example but didn't replace it. Some things I might want to know:
    - What happens when you include or omit a prefix? (Does that prefix every combined tag, or just one?)
    - What happens when you omit the optional `customFields` param?
    - Etc.
- An array of what? ([`@param {string[]}`](http://usejsdoc.org/tags-param.html))

<sup>†</sup> Sidenote, JSDoc has an [`@example`](http://usejsdoc.org/tags-example.html) tag that's useful for use examples which we could use later to generate internal docs.